internal/auth/auth_test.go

```diff
@@ -19,11 +19,12 @@ import (
 
 var _ = Describe("Auth users", func() {
 	var authService *auth.AuthService
-	var fake = &authfakes.FakeSecretInterface{}
-
-	rand.Seed(time.Now().UnixNano())
+	var fake *authfakes.FakeSecretInterface
 
 	BeforeEach(func() {
+		rand.Seed(time.Now().UnixNano())
+
+		fake = &authfakes.FakeSecretInterface{}
```

Comment:
This fixes #1455 ? If yes, should be mentioned as such in the PR description to ensure auto-close of ticket.

Reply:
Edited PR description! :+1: